ENIGMAsystem/SHELL/Graphics_Systems/Direct3D9/DX9model.cpp

```diff
@@ -303,26 +303,26 @@ bool d3d_model_load(int id, string fname)
 
 void d3d_model_draw(int id) // overload for no additional texture or transformation call's
 {
-    meshes[id]->Draw();
+  meshes[id]->Draw();
```

Comment:
we need a bounds check here for debug mode perhaps overload the [] operator?

Reply:
Right, but perhaps you should hold this thought until #1125 so this pr doesn't run astray. There's a lot of places this is missing, better to go at it in one go. I don't think I'll look into this until a next pr.

Reply:
My only question is why you fixed the indents and didn't move the curly braces.

Reply:
Because I intend to also deduplicate the model class in the same way I am deduplicating tiles. They will both use the underlying vertex functions I believe, but at the very least this function is going in general.